src/main/java/com/shapesecurity/shift/reducer/CloneReducer.java

```diff
@@ -131,7 +131,7 @@ public Expression reduceCompoundAssignmentExpression(@NotNull CompoundAssignment
     @NotNull
     @Override
     public MemberExpression reduceComputedMemberExpression(@NotNull ComputedMemberExpression node, @NotNull Node object, @NotNull Node expression) {
-        return new ComputedMemberExpression((Expression) object, (ExpressionSuper) expression);
+        return new ComputedMemberExpression((Expression) expression, (ExpressionSuper) object);
```

Comment:
But I thought you wanted the parameters in the order they appear in the grammar.